aws-redshift-cluster/template.yml

```diff
@@ -5,7 +5,7 @@ Description: AWS SAM template for the AWS::Redshift::Cluster resource type
 Globals:
   Function:
     Timeout: 180  # docker start-up times can be long for SAM CLI
-    MemorySize: 256
+    MemorySize: 2024
```

Comment:
Is this a local change for your docker setup?

Reply:
Yes, but Cluster resource has never worked with 256MB memory, so updated the template to avoid errors in future.